synapse/lib/httpapi.py

```diff
@@ -785,3 +785,54 @@ async def get(self):
             return
         resp = await self.cell.getHealthCheck()
         return self.sendRestRetn(resp)
+
+class StormVarsGetV1(Handler):
+
+    async def get(self):
+
+        body = self.getJsonBody()
+        if body is None:
+            return
+
+        varname = str(body.get('name'))
+        defvalu = body.get('default')
+
+        if not await self.reqAuthAllowed(('globals', 'get', varname)):
+            return
+
+        valu = await self.cell.getStormVar(varname, default=defvalu)
+        return self.sendRestRetn(valu)
+
+class StormVarsPopV1(Handler):
+
+    async def post(self):
+
+        body = self.getJsonBody()
+        if body is None:
+            return
+
+        varname = str(body.get('name'))
+        defvalu = body.get('default')
+
+        if not await self.reqAuthAllowed(('globals', 'pop', varname)):
+            return
+
+        valu = await self.cell.popStormVar(varname, default=defvalu)
+        return self.sendRestRetn(valu)
+
+class StormVarsSetV1(Handler):
+
+    async def post(self):
+
+        body = self.getJsonBody()
+        if body is None:
+            return
+
+        varname = str(body.get('name'))
+        varvalu = body.get('value')
```

Comment:
Would it be expected that a set operation without a value param would set the value to 'None' or should they have to specify a value of None to do that?

Reply:
we should probably detect the difference between "unspecified" and "null".